pkg/ssh/forward.go

```diff
@@ -86,3 +99,87 @@ func forward(localConn net.Conn, client *ssh.Client, remoteNetwork, remoteAddr s
 	waitGroup.Add(1)
 	waitGroup.Wait()
 }
+
+func ReversePortForward(
+	ctx context.Context,
+	client *ssh.Client,
+	remoteNetwork, remoteAddr, localNetwork, localAddr string,
+	exitAfterTimeout time.Duration,
+	log log.Logger,
+) error {
+	listener, err := client.Listen(remoteNetwork, remoteAddr)
+	if err != nil {
+		return err
+	}
+	defer listener.Close()
+
+	done := make(chan struct{})
+	defer close(done)
+
+	go func() {
+		select {
+		case <-done:
+		case <-ctx.Done():
+			_ = listener.Close()
+		}
+	}()
+
+	counter := newConnectionCounter(ctx, exitAfterTimeout, func() {
+		log.Fatalf(
+			"Stopping devpod up, because it stayed idle for a while. You can disable this via 'devpod context set-options -o EXIT_AFTER_TIMEOUT=false'",
```

Comment:
That looks very weird, can you do a log.Fatal instead

Reply:
Done